mappings/net/minecraft/item/ItemGroups.mapping

```diff
@@ -1,27 +1,103 @@
 CLASS net/minecraft/unmapped/C_debvfuic net/minecraft/item/ItemGroups
 	FIELD f_aydfrtnw SPAWN_EGGS Lnet/minecraft/unmapped/C_lnokcayd;
 	FIELD f_cfeykoie COLORED_BLOCKS Lnet/minecraft/unmapped/C_lnokcayd;
-	FIELD f_edxpgotn TOOLS Lnet/minecraft/unmapped/C_lnokcayd;
+	FIELD f_edxpgotn TOOLS_AND_UTILITIES Lnet/minecraft/unmapped/C_lnokcayd;
 	FIELD f_fhmjcumw INGREDIENTS Lnet/minecraft/unmapped/C_lnokcayd;
-	FIELD f_hxatrcot SEARCH Lnet/minecraft/unmapped/C_lnokcayd;
+	FIELD f_hxatrcot SEARCH_ITEMS Lnet/minecraft/unmapped/C_lnokcayd;
 	FIELD f_jebcxmjt ITEM_GROUPS Ljava/util/List;
-	FIELD f_kzghjzzi HOTBAR Lnet/minecraft/unmapped/C_lnokcayd;
+	FIELD f_kzghjzzi SAVED_HOTBARS Lnet/minecraft/unmapped/C_lnokcayd;
 	FIELD f_lihoatrs ENABLED_FEATURES Lnet/minecraft/unmapped/C_czxxrbcp;
 	FIELD f_pnoyoezv BUILDING_BLOCKS Lnet/minecraft/unmapped/C_lnokcayd;
-	FIELD f_pzahtrqp OP Lnet/minecraft/unmapped/C_lnokcayd;
+	FIELD f_pzahtrqp OPERATOR_UTILITIES Lnet/minecraft/unmapped/C_lnokcayd;
 	FIELD f_qapwqnww COMBAT Lnet/minecraft/unmapped/C_lnokcayd;
-	FIELD f_qcxervyf FOOD_AND_DRINK Lnet/minecraft/unmapped/C_lnokcayd;
-	FIELD f_sdiyzpyx NATURAL Lnet/minecraft/unmapped/C_lnokcayd;
-	FIELD f_yqelfgst REDSTONE Lnet/minecraft/unmapped/C_lnokcayd;
-	FIELD f_yqiiybfa INVENTORY Lnet/minecraft/unmapped/C_lnokcayd;
-	FIELD f_zjfnhgwd FUNCTIONAL Lnet/minecraft/unmapped/C_lnokcayd;
+	FIELD f_qcxervyf FOOD_AND_DRINKS Lnet/minecraft/unmapped/C_lnokcayd;
+	FIELD f_sdiyzpyx NATURAL_BLOCKS Lnet/minecraft/unmapped/C_lnokcayd;
+	FIELD f_tiophopq showOperatorItems Z
```

Comment:
Hm, while I use `op` on the builders, I do wonder if I should switch to `showOperatorItems`

Reply:
I would say yes, `op` isn't immediately clear if you're unfamiliar with the code

Reply:
oh heck, I did forget to mention about a third option: `showOpItems` maybe? Because yeah, I do see `op` being used everywhere; `operator` is more obvious on first glance though; Maybe `showOperatorItems` there and `showOpItems` for the args? aaaaaaaa

Reply:
I think either name is good! I'd try to keep them unified though.
my personal preference is `operator` over `op`.

Reply:
fair enough